packages/docs/docs/usage/layout-grid.md

```diff
@@ -0,0 +1,1142 @@
+  +--------------------------+  +--------------------------+  +--------------------------+
+  | Polygon                v |  | 3                        |  |                          |
+  +--------------------------+  +--------------------------+  +--------------------------+
+```
+
+As mentioned before, another theme's grid system may render the above example differently.
+
+### ui:condition
+
+The `ui:condition` "syntax" represents conditional rendering of data in the grid. The condition has four required
+properties, `field`, `value`, `operator` and `children`. Let's talk about each in turn.
+
+- `field`: The dotted-path name of a leaf-level field that makes up the left-side of the condition, used to get data from the `formData`
+- `value`: A single value or a list of values that make up the right-side of the condition
+- `operator`: One of the following three options that controls how the left and right sides of the condition are compared
+  1. `all`: This operator will pass when the right-side and left-side contains all the same value(s)
+  2. `some`: This operator will pass when the right-side and left-side contain as least one value in common
+  3. `none`: This operator will pass when the right-side and left-side do not contain any values in common
+- `children`: The list of `ui:row`, `ui:col` or `ui:columns` definitions to display when the condition is true
+
+In the example above, when the `favorite.shape` field value is `line` then the `favorite.length` field is displayed in
+the grid spanning 4 columns. When it is `circle`, then `favorite.radius` is displayed in the grid spanning 4 columns.
+When it is `polygon`, then `favorite.sides` and `favorite.length` are displayed in the grid, each spanning 4 columns.
+
+We leave it up to the reader to create other examples that use different `operator`s.
+
+## Other capabilities of the LayoutGridField
+
+There are a few more small features that the `LayoutGridField` supports. First, there is a `FormContext` named lookup
+capability that allows one to provide a name-mapped set of strings, components and functions in the `FormContext` inside
+of a well-known location, `layoutGridLookupMap` and then reference those names within the `uiSchema`. Second, the grid
+rendering code recognizes an alternate "syntax" for the objects in `children` arrays in order to display any random
+React element within a grid. Below are examples of them both.
+
+### Named lookup support for className
+
+Since some themes, like `mui`, support CSS-in-JS class names, one cannot hard-code the actual `className` in the
+`uiSchema` easily. That is where the name-mapped information in the `FormContext` comes into play. Let's assume that
+you are using a theme like `mui` and you need to style a `Grid` using CSS-in-JS you could do something like the following:
+
+```json uiSchema.json
+{
+  "ui:field": "LayoutGridField",
+  "ui:layoutGrid": {
+    "ui:row": {
+      "className": "GridStyle anotherClass",
```

Comment:
As a convention, I wonder if we should suggest giving the 'placeholder' class name to be replaced a prefix or something like that, so it's clear that it's not a real classname

Reply:
In the examples? So you mean `GridStyle-REPLACE_ME`?